modules/core_project_factory/main.tf

```diff
@@ -487,3 +487,17 @@ resource "google_project_iam_member" "gke_host_agent" {
     module.project_services,
   ]
 }
+
+/******************************************
```

Comment:
Let's move this invocation into the respective top-level modules.

Reply:
Moved `budget` to be called from the root module directly